CMakeLists.txt

```diff
@@ -9,9 +9,20 @@ if(${CMAKE_VERSION} VERSION_GREATER_EQUAL 3.24)
     cmake_policy(SET CMP0135 NEW)
 endif()
 
+if(${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.14.0")
+	cmake_policy(SET CMP0083 NEW)
+	include(CheckPIESupported)
+	check_pie_supported()
+endif()
```

Comment:
file has
```cmake
cmake_minimum_required(VERSION 3.20)
```
According to https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html#policy-settings the policy will be set to `NEW`.

We should only need the other part of the diff.